pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -22,6 +22,10 @@
 , pkg-config
 }:
 
+assert lib.assertMsg
+  (lib.count lib.id [openclSupport openblasSupport rocmSupport] == 1)
+  "When calling llama-cpp exactly one of the options openclSupport, openblasSupport and rocmSupport should be true!";
```

Comment:
nitpick: I think the wording could be improved: `llama-cpp: exactly one  of openclSupport, openblasSupport and rocmSupport should be enabled`